web/src/components/Content.tsx

```diff
@@ -21,6 +21,7 @@ interface ContentProps {
   overrides?: { [key: string]: { ({ children }: { children: string[] }): ReactElement } };
   onClick?: (url?: string) => void;
   customComponents?: Record<string, ReactElement>;
+  overWriteLinkStyle?: string;
```

Comment:
pebble, have you thought about using `className?: string;` on line 19 instead of `overWriteLinkStyle`?

Reply:
I think this makes sense. We should be able to achieve the same thing with the `className` prop here.

Reply:
done